docs/source/guides/orchestrator_transcoder_gpu.rst

```diff
@@ -0,0 +1,123 @@
+How To Run An Orchestrator With GPU Transcoding
+===============================================
+
+Orchestrators can use GPUs to transcode streams.
+
+Pre-requisites
+**************
+
+Before proceeding with this guide:
+
+- Make sure you followed the :doc:`livepeer installation instructions <../installation>`
+- Make sure you are in the active orchestrator set. See the :doc:`activation guide <../transcoding>`
+- Make sure you have an Ethereum JSON-RPC URL. See the :doc:`Ethereum node guide <../quickstart>`
+- Make sure you have a `Nvidia driver <https://www.nvidia.com/Download/index.aspx>`_ installed on your machine
+- Make sure you have Nvidia GPUs accessible on your machine
+    - See this `list of tested GPUs <https://github.com/livepeer/wiki/blob/master/GPU-SUPPORT.md>`_
```

Comment:
While I know that this link references the Nvidia GPU NVENC support matrix , I'm wondering whether we should add it here as well 

i.e. 

- List of supported Nvidia GPUs 

Thinking as a potential user here, might glance at the table and decide oh my GPU is not in it so it's not supported rather than simply not tested. 

I'll leave it up to you

Reply:
Fair point. Could change the text from "list of tested GPUs" to "GPU support reference" which is a bit more accurate anyway since that document highlights both supported GPUs *and* tested GPUs.